vaadin-spring/src/main/java/com/vaadin/flow/spring/security/JwtSplitCookieService.java

```diff
@@ -0,0 +1,125 @@
+package com.vaadin.flow.spring.security;
```

Comment:
The license header and JavaDoc are missing. 
Also, could we have some unit tests for this class? Looks this is the core of the jwt-based authentication.